components/footer.tsx

```diff
@@ -4,16 +4,18 @@ import parse from 'html-react-parser';
 import { onEntryChange } from '../contentstack-sdk';
 import { getFooterRes } from '../helper';
 import Skeleton from 'react-loading-skeleton';
+import { FooterProps, Entry, Links } from "../typescript/layout";
 
-export default function Footer({ footer, entries }) {
-  const [getFooter, setFooter] = useState(footer);
+export default function Footer({ footer, entries }: {footer: FooterProps, entries: Entry}) {
 
-  function buildNavigation(ent, ft) {
+  const [getFooter, setFooter] = useState(footer);
+  
+  function buildNavigation(ent: Entry, ft: any) {
```

Comment:
footer response should not be any & can be provided as an object(refer to type created for footer)